rlbot/managers/match.py

```diff
@@ -33,6 +33,13 @@ def __init__(
         self.rlbot_interface: SocketRelay = SocketRelay("")
         self.rlbot_interface.packet_handlers.append(self._packet_reporter)
 
+    def __enter__(self) -> 'MatchManager':
+        self.ensure_server_started()
```

Comment:
`ensure_server_started` is automatically called by `start_match`, so it's not needed here

Reply:
Why it is a problem to have it here too?